Update Contributing docs to include guidance around resolving comment… (#3084)
openenclave/openenclave: docs/Community/Contributing.md

- **DO** ensure your correct name and email are on each commit.
- **DO** address PR feedback in an additional commit(s) rather than amending the
  existing commits, and only rebase/squash them when necessary. This makes it

>   easier for reviewers to track changes. After addressing comments, mark as resolved 
>   comments where you did exactly what the reviewer asked, and leave it up to the
>   reviewer to mark anything else as resolved.

Reviewer: ```suggestion
  easier for reviewers to track changes.
- **DO** resolve only the comments that have been addressed exactly as requested by a
  reviewer. Leave it up to the reviewer to mark anything else as resolved.
```
